.github/ISSUE_TEMPLATE/translation.yml

```diff
@@ -0,0 +1,35 @@
+name: Translation
+description: Request a translation
+title: ":twisted_rightwards_arrows: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We appreciate you taking the time to fill out this translation request!
+  - type: input
+    id: word
+    attributes:
+      label: What word do you want to translate (or see translated)?
+    validations:
```

Comment:
Not sure if this is already routed in the validation but it would be cool if this could generate a dropdown all defined words then the "what language from" could share alll languages it exists in?

Definitely a future enhancement tho if big lift!

Reply:
Oh, that's interesting! I think we could do this if we had more control over the form, but the API GitHub provides currently is pretty static.

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

Reply:
We could explore directing folks to use a Typeform form for definitions and translations. This would make the contribution process friendlier to non-GH users. We could use a bit of automation to then create issues from form submissions. I believe I have this captured as something to do in the future, I'll verify and if not add to roadmap.

Reply:
I love this plan to use a separate more user-friendly, non-github-requiring solution. Adding it to future roadmap sounds great to me as an enhancement.